lib/models/dishes.rb

```diff
@@ -5,7 +5,9 @@ class Dish
   property :id, Serial, key: true
   property :name, String, length: 128
   property :menu_id, Serial, key: true
+  property :category_id, Serial, key: true
```

Comment:
Why are you explicitly adding a `category_id` property to this model? same with `menu_id` when you have a `belongs_to` for both the `menu` and the `category`

The moment you have `belongs_to :something` on your model, a field `something_id` is automatically created on that table.